tests/test_external_account.py

```diff
@@ -289,6 +289,7 @@ def test_valid_token_url_shall_pass_validation(self):
             "https://us-east-1-sts.googleapis.com",
             "https://US-WEST-1-sts.googleapis.com",
             "https://us-west-1-sts.googleapis.com/path?query",
+            "https://sts-us-east-1.p.googleapis.com",
```

Comment:
Add valid test case for url with /path added on end

Reply:
Also add test cases for invalid tests - 
            "https://sts.pgoogleapis.com",
            "https://p.googleapis.com",
            "https://sts.p.com",
            "http://sts.p.googleapis.com",
            "https://xyz-sts.p.googleapis.com",
            "https://sts-xyz.123.p.googleapis.com",
            "https://sts-xyz.p1.googleapis.com",
            "https://sts-xyz.p.foo.com",
            "https://sts-xyz.p.foo.googleapis.com");

Reply:
Make sense